### PR TITLE
cmd/goexports,stdlib: parallelize symbols generation for stdlib

### LINKED ISSUE #314: generate: speed-up symbols generation via goexports
on my machine (8procs, 64G RAM, SSDs), running `go generate` or `make generate` takes quite some time.
it seems the bulk of the time is taken by generating the symbols for the whole `stdlib`:

```
$> time go generate ./stdlib

real	0m24.299s
user	0m59.147s
sys	0m4.024s
```

one could probably refactor a bit the symbols generation by giving the whole list of stdlib packages to wrap to `goexports` and, within `goexports`, launch a bunch of goroutines to process each package.

applying that suggestion, I came up with:

```
$> time go generate ./stdlib

real	0m9.747s
user	1m3.719s
sys	0m4.642s
```

interested in a PR?

### PR DESCRIPTION
**Pull request by sbinet**
Fixes #314.

### CODE REVIEW

**Comment by sbinet** [comment]
needs #311.

File: cmd/goexports/goexports.go [hunk 1]
```diff
@@ -212,15 +224,17 @@ func genContent(dest, pkgName string) ([]byte, error) {
...
-		"BuildTags": buildTags,
+		"Dest":        dest,
+		"Imports":     imports,
+		"PkgName":     pkgName,
+		"Val":         val,
+		"Typ":         typ,
+		"Wrap":        wrap,
+		"BuildTags":   buildTags,
+		"License":     license,
+		"BlankImport": len(val)+len(typ)+len(wrap) == 0,
```
[the remark this answers is not in the document]
Comment (sbinet) [reply cmd/goexports/goexports.go]: see #326.

File: stdlib/generate.go [hunk 2]
```diff
@@ -0,0 +1,176 @@
+// +build ignore
+
+// generate symbols to provide access to Go stdlib (https://golang.org/pkg).
+package main
```
[the remark this answers is not in the document]
Comment (sbinet) [reply stdlib/generate.go]: done.

File: cmd/goexports/goexports.go [hunk 3]
```diff
@@ -33,26 +35,31 @@ import (
 	"strconv"
 	"strings"
 	"text/template"
+
+	"golang.org/x/sync/errgroup"
```
[the remark this answers is not in the document]
Comment (sbinet) [reply cmd/goexports/goexports.go]: done: use `sync.WaitGroup`.

**Comment by ldez** [comment]
As I say, I'm working on the goexports command to add a smarter generation by platform.
Use parallelism is a good idea but I will change the core of goexports.
Then we will wait until I finished the smarter generation.

**Comment by sbinet** [comment]
ok. alternatively: we could make `goexports` smarter and recognize (like `go list` does) the "meta" package `stdlib` and have it automatically generate wrappers for all the packages that are in `stdlib`.

**Comment by ldez** [comment]
I do not understand why you close this PR. We are really interested in this PR.

**Comment by ldez** [comment]
If it's a problem, I can review this PR before creating mine, I try to do my best.

**Pull request closed without merging**